### PR TITLE
Markup fixes

### PR DESCRIPTION
**Pull request by max-nes**

### CODE REVIEW

File: src/i18n/en.json [hunk 1]
```diff
@@ -185,7 +185,7 @@
     },
 
     "footer": {
-      "copyright": "2018 © Distributed Lab.",
+      "copyright": "2019 © Distributed Lab.",
```
Comment (quinn71) [inline src/i18n/en.json]: dynamically calculate year in the component?

File: src/vue/navigation/Sidebar.vue [hunk 2]
```diff
@@ -166,38 +150,71 @@ export default {
...
-  color: $col-sidebar-text;
+  flex-direction: column;
 
-  &.router-link-active {
-    background-color: $col-sidebar-active-elem-background;
-    color: $col-sidebar-active-elem-text;
+  @include respond-to-custom($sidebar-hide-bp) {
+    opacity: 1;
+    width: $sidebar-width;
+    background-color: $col-sidebar-background-media-small !important;
```
Comment (quinn71) [inline src/vue/navigation/Sidebar.vue]: comment on why do we use important

File: src/vue/pages/Signup.vue [hunk 3]
```diff
@@ -10,39 +10,43 @@
...
 
     <div class="auth-page__content">
-      <signup-form
-        v-if="!recoveryKeypair"
-        :submit-event="'submit'"
-        @submit="handleChildFormSubmit"
-      />
+      <template v-if="!recoveryKeypair">
+        <signup-form
+          v-if="!recoveryKeypair"
```
Comment (quinn71) [inline src/vue/pages/Signup.vue]: you have the same `v-if` on `template`

File: src/vue/pages/_auth-page.scss [hunk 4]
```diff
@@ -20,6 +20,7 @@
   // TODO: remove !important once md will be completely out
```
Comment (quinn71) [inline src/vue/pages/_auth-page.scss]: it seems we can resolve this TODO now

File: src/i18n/en.json (hunk 1 above)
Comment (max-nes) [reply src/i18n/en.json]: added `currentYear` computed poperty

File: src/vue/pages/Signup.vue (hunk 3 above)
Comment (max-nes) [reply src/vue/pages/Signup.vue]: My bad, removed

File: src/vue/pages/_auth-page.scss (hunk 4 above)
Comment (max-nes) [reply src/vue/pages/_auth-page.scss]: Removed the comment and `!important`

File: src/vue/navigation/Sidebar.vue (hunk 2 above)
Comment (max-nes) [reply src/vue/navigation/Sidebar.vue]: they’re redundant mostyl. removed all the other redundant importants

**Pull request merged**